nihruk/design-system
# Refactor the Search Filter into the Accordion component

The Search Filter component is rather specific, but also built on top of the Bootstrap Accordion component. I propose we refactor ours to be a generic Accordion component, so it can see wider usage.

## This PR
- Renames the component
- Adds a Nunjucks template that also serves to document the allowed variations

## How to test
- Navigate to https://design-system-git-accordion-nihruk.vercel.app/component/accordion
- Confirm the documentation and usage is correct
